plugins/mission/mission.h

```diff
@@ -25,25 +38,97 @@ class Mission
         UNKNOWN
     };
 
+    /**
+     * Returns a human-readable English string for an Mission::Result.
+     */
     static const char *result_str(Result result);
 
+    /**
+     * Callback type for async mission calls.
+     */
     typedef std::function<void(Result)> result_callback_t;
 
-
+    /**
+     * Sends a vector of mission items to the device (asynchronous).
+     *
+     * The mission items are uploaded to a drone and can then be started and executed even if a
+     * connection is lost.
+     *
+     * @param mission_items reference to vector of mission items.
+     * @param callback callback to receive result of this request
+     */
     void send_mission_async(const std::vector<std::shared_ptr<MissionItem>> &mission_items,
                             result_callback_t callback);
 
+    /**
+     * Starts the mission (asynchronous).
+     *
+     * This needs the mission to be sent beforehand using send_mission_async.
```

Comment:
Perhaps make as a "Note". Also make the send_mission_async() a link.

So: 
```
Note that the mission must be uploaded to the vehicle using  `send_mission_async()` before this method is called.
```

Reply:
thx.